util/cliutil/earthlydir.go

```diff
@@ -38,18 +38,23 @@ func makeEarthlyDir() (string, error) {
 	return earthlyDir, nil
 }
 
-// DetectHomeDir returns the home directory of the current user, an additional sudoUser
-// is returned if the user is currently running as root
-func DetectHomeDir() (homeDir string, sudoUser *user.User, err error) {
-	u, err := currentNonSudoUser()
+// DetectHomeDir returns the home directory of the current user, together with
```

Comment:
can you expand this comment to mention what happens when `SUDO_USER` is set?

Reply:
Done